### ISSUE TITLE
some commands fail when there is a space in the filename

### ISSUE THREAD

**Issue by joshrabinowitz** [sobolevn/git-secret#226]
Try manipulating a file like `file with spaces.txt` with git-secret.

See branch displaying problems here:
https://github.com/joshrabinowitz/git-secret/pull/1

-- Problems are in 'changes', 'hide', and 'remove' commands.

Discovered after fixing #135, "some commands fail when the current directory's path includes spaces"

**Comment by joshrabinowitz** [comment]
Here are the particular test failures:

````
 ✗ run 'changes' with one file changed
   (in test file /Users/joshr/gitsrc/git-secret/tests/test_changes.bats, line 39)
     `[ "$status" -eq 0 ]' failed
 ✓ run 'changes' with source file missing
 ✗ run 'changes' with hidden file missing
   (in test file /Users/joshr/gitsrc/git-secret/tests/test_changes.bats, line 58)
     `rm "$encrypted_file"' failed
   rm: local.secret: No such file or directory
 ✗ run 'changes' with one file changed (with deletions)
   (in test file /Users/joshr/gitsrc/git-secret/tests/test_changes.bats, line 71)
     `[ "$status" -eq 0 ]' failed
 ✗ run 'changes' without changes
   (in test file /Users/joshr/gitsrc/git-secret/tests/test_changes.bats, line 85)
     `[ "$status" -eq 0 ]' failed
 ✗ run 'changes' with multiple files changed
   (in test file /Users/joshr/gitsrc/git-secret/tests/test_changes.bats, line 97)
     `[ "$status" -eq 0 ]' failed
 ✗ run 'changes' with multiple selected files changed
   (in test file /Users/joshr/gitsrc/git-secret/tests/test_changes.bats, line 120)
     `[ "$status" -eq 0 ]' failed
````

````
 ✗ run 'hide' with '-m'
   (in test file /Users/joshr/gitsrc/git-secret/tests/test_hide.bats, line 76)
     `[ "$status" -eq 0 ]' failed
 ✗ run 'hide' with '-m' twice
   (in test file /Users/joshr/gitsrc/git-secret/tests/test_hide.bats, line 95)
     `[ "$status" -eq 0 ]' failed
````

````
 ✗ run 'remove' normally
   (in test file /Users/joshr/gitsrc/git-secret/tests/test_remove.bats, line 43)
     `[ "$status" -eq 0 ]' failed
 ✗ run 'remove' with multiple arguments
   (in test file /Users/joshr/gitsrc/git-secret/tests/test_remove.bats, line 68)
     `[ "$status" -eq 0 ]' failed
 ✓ run 'remove' with slashes in filename
 ✗ run 'remove' with '-c'
   (in test file /Users/joshr/gitsrc/git-secret/tests/test_remove.bats, line 116)
     `[ "$status" -eq 0 ]' failed
````

**Comment by joshrabinowitz** [comment]
The only issues remaining are regarding `git-secret remove`. I have fixes worked up for the `hide` and `changes` issues.

**Comment by joshrabinowitz** [comment]
These issues are all resolved in #239
